Backend/Controllers/OrdersController.cs

```diff
@@ -103,6 +103,22 @@ public async Task<IActionResult> CreateOrder([FromBody] CreateOrderDTO dto)
             }
         }
 
+        [HttpPost("{id}/approve")]
+        [Authorize(Roles = "Pharmacy")]
+        public async Task<IActionResult> ApproveOrder(int id)
+        {
+            var user = await GetCurrentUser();
+            var order = Context.Orders.FirstOrDefault(o => o.Id == id);
+
+            if (order == null) return ApiNotFound(ApiErrorSlug.ResourceNotFound, "order");
```

Comment:
Maybe we could extract "order" string literal to constant because it used in another method as well?

Reply:
Extracted